CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to `ord-schema`
+
+Thanks for helping out! What would you like to do?
+
+* [Report an issue](#report-an-issue)
+* [Suggest an improvement](#suggest-an-improvement)
+* [Make a change](#make-a-change)
+
+## Report an issue
+
+Sorry to hear that you're having trouble! Please [file a bug report](https://github.com/Open-Reaction-Database/ord-schema/issues/new?assignees=&labels=bug&template=bug_report.md&title=) to tell us about the problem.
+
+## Suggest an improvement
+
+Great idea! Please [create a feature request](https://github.com/Open-Reaction-Database/ord-schema/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=) to let us know what you'd like to see.
+
+## Make a change
+
+Excellent! There are a few steps you'll need to follow to get ready to submit changes for review:
+
+1. Make sure your changes are consistent with the [goals](https://ord-schema.readthedocs.io/en/latest/overview.html#goals)
+   and [non-goals](https://ord-schema.readthedocs.io/en/latest/overview.html#non-goals) of the database. If your changes
+   address a long-term need we may ask that it be deferred until sometime in the future. In particular, please avoid
+   changes that significantly increase the complexity of creating or using the data.
+1. Make your changes in a [fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-forks)
+   of the `ord-schema` repository. You can work on GitHub or on your local machine:
+   
+   ### GitHub
+   
+   1. Create a fork of `ord-schema` by clicking the "Fork" button at the top-right of this page. 
+      Note that you only need to do this once---all of your changes can use the same fork.
+   1. Follow the [GitHub flow](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/github-flow)
+      to make changes and send a pull request to the official repository.
+   
+   ### Local machine
+   
+   > :warning: Working on your local machine requires familiarity with
+   > [Git](https://git-scm.com/) or [GitHub Desktop](https://desktop.github.com/).
+
+   1. Create a fork of `ord-schema` and clone it to your local machine 
```

Comment:
Perhaps "Clone your fork of `ord-schema` onto your local machine". Your link to how to fork a repo can be moved to the instruction above ("1. Create a fork of `ord-schema` by clicking the "Fork" button at the top-right of this page. ")

Reply:
My concern here is that people who only want to use GitHub will go to that link and have trouble with all the cloning instructions---which make up most of that doc. The doc doesn't actually tell you how to clone a repo that isn't their example :)

Maybe I'll just copy the "click Fork" down here as well?